src/guiguts/mainwindow.py

```diff
@@ -876,7 +887,4 @@ def statusbar() -> StatusBar:
     return MainWindow.statusbar
 
 
-def messagelog() -> MessageLog:
-    """Return the single MessageLog object"""
-    assert MainWindow.messagelog is not None
-    return MainWindow.messagelog
+logger = logging.getLogger(__package__)
```

Comment:
The usual convention is to have this at the top of the file after the imports and before the class definitions. I'd probably put it before the constants but both before and after make sense to me.